README.md

```diff
@@ -33,7 +33,7 @@
 * Import/Export wallet accounts (NEP6 Standard)
 * View balance
 * View prices for GAS and NEO in multiple currencies
-* Send GAS, NEO, and certain NEP5 tokens (RPX, DBC, QLC)
+* Send GAS, NEO, and certain NEP5 tokens (RPX, DBC, QLC, IAM, ONT and THOR)
```

Comment:
Thanks for the contribution!  I would suggest we remove the list of tokens (since any NEP5 token can be added by users now) and just change this line to:

> Send GAS, NEO, and any NEP5 token

Reply:
Hi @mhuggins that would be a better pick indeed!